frame/evm-system/src/lib.rs

```diff
@@ -182,16 +182,30 @@ impl<T: Config> StoredMap<<T as Config>::AccountId, <T as Config>::AccountData>
 		k: &<T as Config>::AccountId,
 		f: impl FnOnce(&mut Option<<T as Config>::AccountData>) -> Result<R, E>,
 	) -> Result<R, E> {
-		let mut maybe_account_data = if Self::account_exists(k) {
-			Some(Account::<T>::get(k).data)
+		let (mut some_data, was_providing) = if Self::account_exists(k) {
```

Comment:
```suggestion
		let (mut maybe_account_data, was_providing) = if Self::account_exists(k) {
```

The `some_data` is kind of misleading, not sure you decided to switch from `maybe_account_data` but the old name is better imo

Reply:
https://github.com/humanode-network/frontier/pull/70/commits/f2eea5392775b795628fb3a93216fb654b0b739f